rasa_core/policies/keras_policy.py

```diff
@@ -55,10 +67,21 @@ def __init__(self,
 
         self.current_epoch = current_epoch
 
+    @staticmethod
+    def _load_tf_config(config: Dict[Text, Any]) -> None:
+        tf_config = tf.ConfigProto(
+            device_count={'CPU': config['device_count']},
+            inter_op_parallelism_threads=config['inter_op_threads'],
+            intra_op_parallelism_threads=config['intra_op_threads'],
+            gpu_options={'allow_growth': config['allow_growth']}
+        )
+        return tf_config
+
     def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
         config = copy.deepcopy(self.defaults)
         config.update(kwargs)
 
+        self.config = config
```

Comment:
similar to embed policy, let's load tf config here into `self._tf_config`

Reply:
embed policy is easy to add `self._tf_config`. but keras_policy if I set `self._tf_config = self._load_tf_config(config)`, it cannot persist self._tf_config as json format. Now I persist the config parameter in `keras_policy.json`

Reply:
I see the problem.

If it is not possible in `keras`, let's make `config` a class variable and persist it in both policies. then load `tf_config` from there. It is better to be consistent across different policies

Reply:
Also, I think it makes sense to create `tf_defaults` in `policy.py`, then merge two defaults together in `load_param` method. What do you thing?

Reply:
ok, this idea I try this weekend. It is better to be consistent across different pilocies.

Reply:
I resolved it and pushed the code already.